services/github/github-release.service.js

```diff
@@ -39,22 +92,106 @@ module.exports = class GithubRelease extends GithubAuthService {
     }
   }
 
-  static render({ version, isPrerelease }) {
-    return {
-      message: addv(version),
-      color: isPrerelease ? 'orange' : 'blue',
+  static render({ version, sort, isPrerelease }) {
+    let color = 'blue'
+    color = sort === 'semver' ? versionColor(version) : color
+    color = isPrerelease ? 'orange' : color
+    return { label: 'release', message: addv(version), color }
+  }
+
+  async fetchReleases({ user, repo }) {
+    return this._requestJson({
+      url: `/repos/${user}/${repo}/releases`,
+      schema: releaseInfoArraySchema,
+      errorMessages: errorMessagesFor('no releases or repo not found'),
+    })
+  }
+
+  static getLatestRelease({ releases, sort, includePrereleases }) {
+    if (sort === 'semver') {
+      const latestRelease = latest(releases.map(release => release.tag_name), {
+        pre: includePrereleases,
+      })
+      const kvpairs = Object.assign(
+        ...releases.map(release => ({ [release.tag_name]: release.prerelease }))
+      )
+      return { tag_name: latestRelease, prerelease: kvpairs[latestRelease] }
```

Comment:
Nice 👍 

Side note, but it might be nice one day if we could have the `latest` helper return the prerelease indicator along with the latest version it found (something like `return { version, prerelease: ... }`  instead of `return version`) to avoid the need for callers to have to make that check themselves